pkgs/by-name/pa/paperless-ngx/package.nix

```diff
@@ -199,6 +193,7 @@ python.pkgs.buildPythonApplication rec {
       djangorestframework
       djangorestframework-guardian2
       drf-writable-nested
+      fido2
```

Comment:
This will come when we update django-allauth to a newer version with the optional dep